src/module/Ui/templates/layout/de/serlo-home.phtml

```diff
@@ -157,69 +158,137 @@ echo $this->doctype();
+                <div class="home-row home-column visible-md visible-lg">
+                    <div class="col-sm-5 col-lg-4 col-lg-offset-2 home-calls-column">
+                        <a href="/mitmachen" class="home-emph-link">
+                            Zur Startseite für AutorInnen <i class="fa fa-arrow-circle-right"></i>
                         </a>
-                        <a href="/21456">
-                            <img src="/assets/images/de/home_logo_wiki.svg" alt="Wikimedia Deutschland"/>
+                    </div>
+                    <div class="col-sm-5 col-lg-4 home-calls-column col-md-offset-1 col-lg-offset-0">
+                        <a href="/jobs" class="home-emph-link">
+                            Jobs und Engagement <i class="fa fa-arrow-circle-right"></i>
                         </a>
                     </div>
-                    <div class="home-column home-col-lg-1"></div>
+                </div>
                 </div>
 
+                <div class="home-row home-principle-row home-column col-md-12 col-lg-6 col-lg-offset-3">
+                    <img src="/assets/images/de/about_serlo.svg">
+                </div>
                 <div class="home-launch-img-row home-row"></div>
-
-                <div class="home-community-wrapper">
-                    <div class="home-community-row home-row">
-                        <div class="home-column home-col-md-3">
-                            <img src="/assets/images/home_participate.svg"/>
-
-                            <h2>Serlo ist ein Wiki:</h2>
-
-                            <p>
-                                Hilf uns Bildung frei verfügbar zu machen und werde Teil der Online-Community von Serlo.
-                                Wir suchen Lehrer und Hobbydidaktiker mit Begeisterung für ihr Fach.
-                            </p>
-                            <img src="/assets/images/home_commons.svg"/><br/>
-                            <a href="/19869" class="home-hide-md home-emph-link">Mitmachen <i class="fa fa-arrow-circle-right"></i></a>
-                        </div>
-                        <div class="home-column home-col-md-3">
-                            <img src="/assets/images/home_donate.svg"/>
-
-                            <h2>Serlo ist eine Non-Profit-Organisation:</h2>
-
-                            <p class="text-center">
-                                Softwareentwicklung, Betreuung der Ehrenamtlichen und Aufbau der Organisation kosten
-                                Geld. Wir freuen uns sehr, wenn Sie Serlo mit einer Spende unterstützen.
-                            </p>
-                            <a href="/21565" class="home-hide-md home-emph-link">Spenden <i class="fa fa-arrow-circle-right"></i></a>
+                <div class="home-calls-wrapper">
+                    <div class="home-row home-column">
+                        <!-- <div class="home-column heart extra-icon col-md-12 col-lg-10 col-lg-offset-2">
```

Comment:
I thought I might need that code later if we decide to put these icons back on top, I'll remove if everyone is fine with how it is now.